bigquery/gcloud/aio/bigquery/bigquery.py

```diff
@@ -282,16 +293,23 @@ async def insert_via_load(
             autodetect: bool = False,
             source_format: SourceFormat = SourceFormat.CSV,
             write_disposition: Disposition = Disposition.WRITE_TRUNCATE,
-            timeout: int = 60) -> Dict[str, Any]:
+            timeout: int = 60,
+            ignore_unknown_values: bool = False,
+            schema_update_options: Optional[List[SchemaUpdateOption]] = None
+        ) -> Dict[str, Any]:
         """Loads entities from storage to BigQuery."""
         if not source_uris:
             return {}
 
+        if not schema_update_options:
```

Comment:
pro-tip: you can also do default values like this with `or`, eg:
```python
body = self._make_load_body(
    source_uris, project, autodetect, source_format, write_disposition,
    ignore_unknown_values, schema_update_options or [],
)
```

Reply:
Yeah, it's okay here, but I think it's generally not the best pattern as it won't allow user to pass a falsey value.

Reply:
... I could say if I actually wrote `if schema_update_options is None` 🤦